plugins/GrapesJsBuilderBundle/Translations/cs/javascript.ini

```
@@ -8,3 +8,4 @@ grapesjsbuilder.dynamicContentBlockLabel="Dynamický obsah"
 grapesjsbuilder.dynamicContentBtnLabel="Uložit"
 grapesjsbuilder.dynamicContentModalTitle="Upravit dynamický obsah"
 grapesjsbuilder.assetManager.noAssets="Nejsou zde žádné <b>assety</b>, přetáhněte sem soubory pro nahrání."
+grapesjsbuilder.buttonBlockLabel="Knoflík"
```

I am not sure If this is regular Czech translation :) @escopecz

---

I have found tlačítka on google translate for string buttons but when I translate for string button it gives Knoflík. So guys if you have idea about what correct translation is for the above language please let me know. I will change and commit the file again. Thanks @kuzmany @escopecz

---

```suggestion
grapesjsbuilder.buttonBlockLabel="Tlačítko"
```
This is much better. "Knoflík" is used to button up a shirt.